puppet/modules/jenkins_node/manifests/packaging/rpm.pp

```diff
@@ -69,20 +62,6 @@
     ensure => present,
```

Comment:
the line above contains `scl-utils`, which we don't need on el8+ either.

Reply:
I also thought about that, but wasn't sure if we should ensure it's absent first.

Reply:
We're about to re-deploy all nodes with el9, so I wouldn't care too much.